### PR TITLE
chore: Follow up on Popover tip refinement

### PR DESCRIPTION
**Pull request by at-susie**
Apply the same refinement of Popover tip, which was done in [this PR](https://github.com/cloudscape-design/components/pull/1916), to other Popovers used in Charts and Breadcrumb

### Description

Related links, issue #, if available: n/a

### How has this been tested?

<details>
   <summary>Review checklist</summary>

_The following items are to be evaluated by the author(s) and the reviewer(s)._

#### Correctness

- _Changes include appropriate documentation updates._
- _Changes are backward-compatible if not indicated, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#public-apis)._
- _Changes do not include unsupported browser features, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#browsers-support)._
- _Changes were manually tested for accessibility, see [accessibility guidelines](https://cloudscape.design/foundation/core-principles/accessibility/)._

#### Security

- _If the code handles URLs: all URLs are validated through [the `checkSafeUrl` function](https://github.com/cloudscape-design/components/blob/main/src/internal/utils/check-safe-url.ts)._

#### Testing

- _Changes are covered with new/existing unit tests?_
- _Changes are covered with new/existing integration tests?_
</details>

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/breadcrumb-group/item/item.tsx [hunk 1]
```diff
@@ -58,7 +61,7 @@ const BreadcrumbItemWithPopover = <T extends BreadcrumbGroupProps.Item>({
               arrow={position => (
                 <div className={clsx(popoverStyles.arrow, popoverStyles[`arrow-position-${position}`])}>
                   <div className={popoverStyles['arrow-outer']} />
-                  <div className={popoverStyles['arrow-inner']} />
+                  <div className={clsx(popoverStyles['arrow-inner'], isVisualRefresh && popoverStyles.refresh)} />
                 </div>
```
Comment (just-boris) [inline src/breadcrumb-group/item/item.tsx:65]: Can we change this code and the one below to use the `<Arrow>` component from the previous PR: https://github.com/cloudscape-design/components/blob/b4da3b8e1a16b4eb746e1f603dce3b49175c633d/src/popover/arrow.tsx#L14-L23

Comment (just-boris) [reply src/breadcrumb-group/item/item.tsx:65]:
```suggestion
                <Arrow position={position} />
```

Comment (at-susie) [reply src/breadcrumb-group/item/item.tsx:65]: That makes sense. I'll do that. Thanks for the suggestion.

**Comment by just-boris** [comment]
Solved in https://github.com/cloudscape-design/components/pull/2085

**Pull request closed without merging**